web-app/src/components/DataTable/Cells/teamsOverview.js

```diff
@@ -0,0 +1,25 @@
+const fullName = {
+  id: 'fullName',
+  Header: 'Full Name',
+  accessor: user => user.members.map( element => element.name ),
+};
+  
+const state = {
+  id: 'state',
+  Header: 'State',
+  accessor: user => user.members.map( element => element.state ),
+};
+  
+const team = {
+  id: 'team',
+  Header: 'Team',
+  accessor: user => user.team,
+};
+  
+  
+export default {
+  fullName,
+  state,
+  team,
+};
+  
```

Comment:
Add a newline at the end of the file. I'd recommend something like Prettier to do this for you when you save.